iotdb-core/datanode/src/main/java/org/apache/iotdb/db/protocol/mqtt/LinePayloadFormatter.java

```diff
@@ -63,7 +64,7 @@ public LinePayloadFormatter() {
   }
 
   @Override
-  public List<Message> format(ByteBuf payload) {
+  public List<Message> format(String topic, ByteBuf payload) {
```

Comment:
Since the topics can be obtained here, I think the logic for setting up the database should be included here

```
        if (useTableInsert) {
          TableMessage tableMessage = (TableMessage) message;
          // '/' previously defined as a database name
          String database =
              !msg.getTopicName().contains("/")
                  ? msg.getTopicName()
                  : msg.getTopicName().substring(0, msg.getTopicName().indexOf("/"));
          tableMessage.setDatabase(database);
```